internal/service/anycast/model_proto_anycast_config.go

```diff
@@ -58,7 +56,7 @@ var ProtoAnycastConfigResourceSchemaAttributes = map[string]schema.Attribute{
 		MarkdownDescription: "The account identifier.",
 	},
 	"anycast_ip_address": schema.StringAttribute{
-		Optional: true,
+		Required: true,
 		PlanModifiers: []planmodifier.String{
 			stringplanmodifier.RequiresReplaceIfConfigured(),
```

Comment:
you can't update IP address?

Reply:
this is not yet addressed.

Reply:
you deleted the description as well, accidental?

Reply:
added back